### PR TITLE
RES-358 - Update information on debug page

### PR DESCRIPTION
**Pull request by vasturiano**
More details on https://wiki.cfops.it/display/RES/Resolver+Debug+Page

Data is extracted from the API calls:
https://is-cf.cloudflareresolve.com/resolvertest
https://is-dot.cloudflareresolve.com/resolvertest
https://is-doh.cloudflareresolve.com/resolvertest
https://1.1.1.1/resolvertest
...
https://1.1.1.1/cdn-cgi/trace
https://(uuid).map.cloudflareresolve.com

Design is still a little bare bones but we can improve the visuals later.

screen shot 2018-06-07 at 6 03 08 pm

### CODE REVIEW

**Comment by dknecht** [comment]
Nice!

**Comment by irtefa** [comment]
🔥

File: source/pages/partials/help.pug [hunk 1]
```diff
@@ -6,34 +6,40 @@ section(data-section="help")
...
+    table.debug-table
+      tr
+        td Connected to Resolver
+        td(data-ref="isCf") Loading...
+      tr
+        td Using DNS over HTTPS (DoH)
+        td(data-ref="isDot") Loading...
+      tr
+        td Using DNS over TLS (DoT)
+        td(data-ref="isDoh") Loading...
```
Comment (vavrusa) [inline source/pages/partials/help.pug]: @vasturiano you're showing DoH result for DoT and vice versa ^

**Comment by vavrusa** [comment]
Looks nice otherwise!

File: source/pages/partials/help.pug (hunk 1 above)
Comment (vasturiano) [reply source/pages/partials/help.pug]: Great catch @vavrusa! I'll fix it.

**Pull request merged**